frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -675,3 +675,142 @@ export const GRAPH_AXIS_SETTINGS = {
   // DEPRECATED" replaced with "label" series setting
   "graph.series_labels": {},
 };
+
+export const SANKEY_DATA_SETTINGS = {
+  ...columnSettings({
+    getColumns: (
+      [
+        {
+          data: { cols },
+        },
+      ],
+      settings,
+    ) => cols,
+    hidden: true,
+  }),
+  "graph._start_filter": {
+    getDefault: ([{ card }]) => {
+      return isDimension
+    },
+    useRawSeries: true,
+  },
+  "graph._destinations_filter": {
+    getDefault: ([{ card }]) => isDimension,
+    useRawSeries: true,
+  },
+  "graph._metric_filter": {
+    getDefault: ([{ card }]) => isMetric,
```

Comment:
since both `graph._destinations_filter` and `graph._metric_filter` don't have any dependencies on the passed `card` unlike this 
https://github.com/metabase/metabase/blob/a48893b44db7905f5e1165e84a0a0032992938f9/frontend/src/metabase/visualizations/lib/settings/graph.js#L93-L97
there's no need to have these 2 settings.

Reply:
removed

Reply:
Not sure if you have pushed the change, but I can still see that `graph._metric_filter` exists.

Reply:
Sorry, it's been removed now.